modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -65,115 +65,59 @@ endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 . Set the `$RELEASE_IMAGE` variable by running the following command:
 +
-[source,terminal] 
+[source,terminal]
 ----
 $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
 endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
+//These conditions could be applied to only what is different (`--cloud=<blah>`) to avoid maintenance difficulties.
```

Comment:
What's this line about?  I expect `--included` to include the cloud (among other parameters) from `--install-config`, without needing folks to set `--cloud` explicitly anymore.

Reply:
oh wow, I didn't realize it was taking care of `--cloud` as well. Handy! I'll need to scrub through for that as well then :) 

That note was for myself to do some reorganizing, which I also now will not need to do anyway :nerd_face: